CHANGELOG.md

```diff
@@ -3,6 +3,35 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v.5.10.0
+
+_Apr 22, 2022_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```

Comment:
@DanailH Would you add a highlight for Row reorder once it's merged?

Reply:
Yes, I will merge it tomorrow. I've updated the docs and will wait for people to check it once more.

Reply:
Great!

Reply:
A gif would be awesome, since we don't have too much content.

Reply:
added